src/components/new-safe/load/steps/Step2/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { Box, Button, Divider, Grid, Typography } from '@mui/material'
+
+import type { StepRenderProps } from '@/components/new-safe/CardStepper/useCardStepper'
+import type { LoadSafeFormData } from '@/components/new-safe/load'
+import layoutCss from '@/components/new-safe/create/styles.module.css'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import ChainIndicator from '@/components/common/ChainIndicator'
+import css from '@/components/new-safe/create/steps/Step3/styles.module.css'
+import EthHashInfo from '@/components/common/EthHashInfo'
+import { useCurrentChain } from '@/hooks/useChains'
+import { useAppDispatch } from '@/store'
+import { useRouter } from 'next/router'
+import { addOrUpdateSafe } from '@/store/addedSafesSlice'
+import { defaultSafeInfo } from '@/store/safeInfoSlice'
+import { upsertAddressBookEntry } from '@/store/addressBookSlice'
+import { LOAD_SAFE_EVENTS, trackEvent } from '@/services/analytics'
+import { AppRoutes } from '@/config/routes'
+import ReviewRow from '@/components/new-safe/ReviewRow'
+
+const LoadSafeStep2 = ({ data, onBack }: StepRenderProps<LoadSafeFormData>) => {
```

Comment:
Not a fan of numeric names, tbh. The name should reflect what the component does, not the position this component is included at.

Reply:
I agree, lets change it for load safe now since the old components are deleted and for create safe with #1404